fluent-package/debian/fluent-package.lintian-overrides

```diff
@@ -4,17 +4,12 @@ fluent-package: script-not-executable
 fluent-package: python-script-but-no-python-dep
```

Comment:
Shouldn't we remove `python-script-but-no-python-dep` too?

```suggestion
```

Reply:
Thanks, fixed now.